docs/docs/uninstall.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Manual Uninstall
+---
+
+------------
+
+**DISCLAIMER**
+_This is a very early version of Cassandra-on-Mesos framework. This document, code behavior, and anything else may change without notice and/or break older installations._
+
+------------
+
+
+# Manual Uninstall on DCOS
```

Comment:
Can we make `#Manual Uninstall` an additional H1 heading at the top and make this and the other heading H2 (i.e. `##`)?